addons/im_livechat/demo/im_livechat_channel/im_livechat_chatbot_session_1.xml

```diff
@@ -5,6 +5,7 @@
             <field name="channel_type">livechat</field>
             <field name="livechat_channel_id" ref="im_livechat_channel_data"/>
             <field name="livechat_operator_id" ref="welcome_bot_operator_partner_demo"/>
+            <field name="livechat_failure">no_failure</field>
```

Comment:
I think it would also be fine to leave some of them wit no `livechat_failure` to simulate what's happening with old database that didn't have the field.